### PR TITLE
Add ua override to remove both app id and version

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205183481775449/f 

### Description
This PR adds a new UA override

### Steps to test this PR

- [x] Build using internal flavour
- [x] Go to fedex.com
- [x] You should a message saying the browser is not compatible
- [x] Go to internal dev settings and override the UA to use no app id and no version
- [x] Go to fedex.com, refresh, you should not see the incompatible browser message
- [x] It make take you a refresh or two for the UA change to happen.

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3410** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3410" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3410?utm_source=stack-comment).

**Review by cmonfortep — APPROVED** [review]
LGTM

File: app/src/internal/java/com/duckduckgo/app/dev/settings/InternalUAInterceptor.kt [hunk 1]
```diff
@@ -46,6 +46,7 @@ class InternalUAInterceptor @Inject constructor(
             UAOverride.CHROME -> userAgent.replace(applicationComponent, "").replace(version, "")
             UAOverride.FIREFOX -> "Mozilla/5.0 (Android 11; Mobile; rv:94.0) Gecko/94.0 Firefox/94.0"
             UAOverride.WEBVIEW -> defaultUserAgent.get()
+            UAOverride.NO_APP_ID_AND_NO_VERSION -> userAgent.replace(applicationComponent, "").replace(version, "")
```
Comment (anikiki) [inline app/src/internal/java/com/duckduckgo/app/dev/settings/InternalUAInterceptor.kt:49]: This is exactly the same as using the CHROME override?

Comment (marcosholgado) [reply app/src/internal/java/com/duckduckgo/app/dev/settings/InternalUAInterceptor.kt]: You are 100% correct. I have changed the PR and just amended the string for chrome to specify it means not using the app id and the version 👍

File: app/src/internal/res/values/donottranslate.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
     <string name="devSettingsScreenUserAgentSelectorTitle">UserAgent</string>
     <string name="devSettingsScreenUserAgentNoAppId">Omit DuckDuckGo AppId</string>
     <string name="devSettingsScreenUserAgentNoVersion">Omit Webview Version</string>
-    <string name="devSettingsScreenUserAgentChrome">Chrome</string>
+    <string name="devSettingsScreenUserAgentChrome">Chrome (no appId and no Version)</string>
```
Comment (anikiki) [inline app/src/internal/res/values/donottranslate.xml:41]: :+1:

**Review by anikiki — APPROVED** [review]
LGTM :+1:

**Pull request merged**